participant/src/round2.rs

```diff
@@ -0,0 +1,135 @@
+use crate::Logger;
+use frost::{
+    keys::KeyPackage,
+    round1::{NonceCommitment, SigningCommitments, SigningNonces},
+    round2::{self, SignatureShare},
+    Error, Identifier, SigningPackage,
+};
+use frost_ed25519 as frost;
+use hex::FromHex;
+use std::io::BufRead;
+
+// #[derive(Debug)]
+pub struct Round2Config {
+    pub message: Vec<u8>,
+    pub signer_commitments: Vec<SigningCommitments>,
+}
+
+// TODO: refactor to generate config
+// TODO: handle errors
+pub fn round_2_request_inputs(
+    signing_commitments: SigningCommitments,
+    input: &mut impl BufRead,
+    logger: &mut dyn Logger,
+) -> Result<Round2Config, Error> {
+    logger.log("=== Round 2 ===".to_string());
+
+    logger.log("Number of signers:".to_string());
+
+    let mut signers_input = String::new();
+
+    input.read_line(&mut signers_input).unwrap();
+
+    let signers = signers_input.trim().parse::<u16>().unwrap();
+
+    logger.log("You will receive a message from the coordinator, please enter here:".to_string());
```

Comment:
I was a bit confused by the prompt (I assumed it was a protocol message with the commitments or something like that, not the message to sign). Here's a suggestion, but feel free to change:

```suggestion
    logger.log("Enter the message to sign (received from the coordinator):".to_string());
```